doc/engineering-reference/src/advanced-surface-concepts/green-roof-model-ecoroof.tex

```diff
@@ -46,7 +46,7 @@ \subsection{Green Roof Model Description}\label{green-roof-model-description}
 
 The energy budget analysis follows the Fast All Season Soil Strength (FASST) model developed by Frankenstein and Koenig for the US Army Corps of Engineers. FASST was developed, in part, to determine the ability of soils to support manned and unmanned vehicles and personnel movement. In order to accomplish this, however, FASST tracks the energy and moisture balance (including ice and snow) within a vegetated soil. It is a one-dimensional model that draws heavily from other plant canopy models including BATS (Dickinson et al.) and SiB (Sellers et al.). We have implemented FASST here with only a few modifications to adapt it for use with a relatively thin soil layer. The sign convention used assumes all heat fluxes are positive when energy is absorbed into the layer.
 
-In the following discussion this energy budget is divided into a budget for the foliage layer (F\(_{f}\)) and a budget for the ground surface (F\(_{g}\)­). The various parameterizations for latent and sensible heat flux are described in some detail and then the equation set is reduced to the simultaneous solution of two equations involving the temperatures of the foliage and ground surface.
+In the following discussion this energy budget is divided into a budget for the foliage layer (F\(_{f}\)) and a budget for the ground surface (F\(_{g}\)-). The various parameterizations for latent and sensible heat flux are described in some detail and then the equation set is reduced to the simultaneous solution of two equations involving the temperatures of the foliage and ground surface.
```

Comment:
It looks like you put the minus sign outside the math mode in this line but on line 65 you kept it inside.  I'm not sweating this detail though, everything else looks great.  I'll make sure it builds and merge.  Thanks!

Reply:
Actually, I am guessing they probably want it to be in the subscript.  I'm going to go ahead and modify this line so the minus is in the subscript like it is in the diff below.  I built the docs locally, no issues.  So after I make that tiny change, I'll merge it in right away so the commit doesn't waste CI resources.